Gem/Code/Source/Automation/ScriptReporter.cpp

```diff
@@ -1081,4 +1091,46 @@ namespace AtomSampleViewer
         }
 
     }
+
+    void ScriptReporter::ExportTestResults()
+    {
+        uint32_t totalAssertCount = 0;
+        uint32_t totalErrorCount = 0;
+        uint32_t totalWarningCount = 0;
+        uint32_t totalScreenshotErrorCount = 0;
+        uint32_t totalScreenshotWarningCount = 0;
+        
+        for (ScriptReport& scriptReport : m_scriptReports)
+        {
+            totalAssertCount += scriptReport.m_assertCount;
+            totalErrorCount += scriptReport.m_generalErrorCount;
+            totalWarningCount += scriptReport.m_generalWarningCount;
+            totalScreenshotErrorCount += scriptReport.m_screenshotErrorCount;
+            totalScreenshotWarningCount += scriptReport.m_screenshotWarningCount;
+        }
+
+        AZStd::string assertLogLine = AZStd::string::format("Asserts: %i \n", totalAssertCount);
+        AZStd::string errorsLogLine = AZStd::string::format("Errors: %i \n", totalErrorCount);
+        AZStd::string warningsLogLine = AZStd::string::format("Warnings: %i \n", totalWarningCount);
+        AZStd::string screenshotErrorsLogLine = AZStd::string::format("Screenshot errors: %i \n", totalScreenshotErrorCount);
+        AZStd::string screenshotWarningsLogLine = AZStd::string::format("Screenshot warnings: %i \n", totalScreenshotWarningCount);
+
+        auto io = AZ::IO::LocalFileIO::GetInstance();
+        AZStd::string projectPath = AZ::Utils::GetProjectPath();
+        AZStd::string exportFileName = "exportedTestResults.txt";
+        AzFramework::StringFunc::Path::Join(projectPath.c_str(), "TestResults/", m_exportTestResultsFolder);
+        AzFramework::StringFunc::Path::Join(m_exportTestResultsFolder.c_str(), "exportedTestResults.txt", exportFileName);
+        AZ::IO::HandleType logHandle;
+        
+        io->CreatePath(m_exportTestResultsFolder.c_str());
+        io->Open(exportFileName.c_str(), AZ::IO::OpenMode::ModeWrite, logHandle);
```

Comment:
probably should wrap the below with `if (io->Open(...))` in the event we failed to open the file (e.g. file in use elsewhere, file permissions issue, etc.)